lib/Common/CommonPal.h

```diff
@@ -119,6 +119,28 @@ inline void DebugBreak()
 // activscp.h
 #define SCRIPT_E_RECORDED                _HRESULT_TYPEDEF_(0x86664004L)
 
+
+typedef
+enum tagBREAKPOINT_STATE
```

Comment:
Script debugging/profiling are deeply embedded in many places (all the entryPoint Asserts, references from .asm...). To completely ifdef out will incur lots of code noise. I asked Sandeep and was told the current script debugging code will largely be reused for future JSRT debugging. That gave me less incentive to cleanly ifdef out all the debugging/profiling code. These enum types/constants are used in internal debugger logic. By adding them here the related code builds. We can remove them when future debugging code is rewritten to use new JSRT debugging types/constants.